src/EFCore.MySql/Migrations/MySqlMigrationsSqlGenerator.cs

```diff
@@ -391,16 +391,10 @@ protected override void Generate(DropSchemaOperation operation, IModel model, Mi
             builder
                 .Append("CREATE SEQUENCE ")
                 .Append(Dependencies.SqlGenerationHelper.DelimitIdentifier(operation.Name, operation.Schema));
```

Comment:
We need a tests to verify the behavior. They should be implemented in [MigrationSqlGeneratorMySqlTest.MySql.cs](https://github.com/PomeloFoundation/Pomelo.EntityFrameworkCore.MySql/blob/master/test/EFCore.MySql.FunctionalTests/MigrationSqlGeneratorMySqlTest.MySql.cs).

Because this feature is supported only by some MariaDB versions, most tests will need to be annotated using `SupportedServerVersionFactAttribute`.
Some might need the opposite, which is `SupportedServerVersionLessThanFactAttribute`.